ggshield/core/dirs.py

```diff
@@ -7,15 +8,15 @@
 APPAUTHOR = "GitGuardian"
 
 
-def get_config_dir() -> str:
+def get_config_dir() -> Any:
     try:
         # See tests/conftest.py for details
         return str(os.environ["GG_CONFIG_DIR"])
     except KeyError:
         return user_config_dir(appname=APPNAME, appauthor=APPAUTHOR)
 
 
-def get_cache_dir() -> str:
+def get_cache_dir() -> Any:
```

Comment:
The changes in these files should not be necessary now that `os.environ` is turned into an `str`.